hw/ip/entropy_src/doc/checklist.md

```diff
@@ -35,24 +35,24 @@ Code Quality  | [LINT_SETUP][]                 | Done        |
 
 Type          | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
-Documentation | [NEW_FEATURES][]        | Progressing |
+Documentation | [NEW_FEATURES][]        | Done        |
 Documentation | [BLOCK_DIAGRAM][]       | Done        |
 Documentation | [DOC_INTERFACE][]       | Done        |
-Documentation | [MISSING_FUNC][]        | Progressing |
-Documentation | [FEATURE_FROZEN][]      | No          |
-RTL           | [FEATURE_COMPLETE][]    | Progressing |
+Documentation | [MISSING_FUNC][]        | Done        |
+Documentation | [FEATURE_FROZEN][]      | Done        |
+RTL           | [FEATURE_COMPLETE][]    | Done        |
```

Comment:
@moidx and/or @cdgori  Can you confirm that the entropy source has all necessary features implemented?

Reply:
health tests and boot and runtime init flows meet expected requirements. 
 
Note on software access:  `FW_OV_CONTROL` will be used to access raw entropy in software and to perform a KAT test of the conditioner. `ENTROPY_DATA` output will still be required to check the KAT output of the conditioner. 

Security hardening will be covered as part of D3 review. Security review is WIP.